README.md

```diff
@@ -6,29 +6,23 @@ Provides standard objects and functions for working with dates and times.
 
 ## Champions
 
--   Maggie Pint ([@maggiepint](https://github.com/maggiepint))
--   Philipp Dunkel ([@pipobscure](https://github.com/pipobscure))
--   Matt Johnson ([@mj1856](https://github.com/mj1856))
--   Brian Terlson ([@bterlson](https://github.com/bterlson))
--   Shane Carr ([@sffc](https://github.com/sffc))
--   Ujjwal Sharma ([@ryzokuken](https://github.com/ryzokuken))
--   Philip Chimento ([@ptomato](https://github.com/ptomato))
--   Jason Williams ([@jasonwilliams](https://github.com/jasonwilliams))
--   Justin Grant ([@justingrant](https://github.com/justingrant))
+- Maggie Pint ([@maggiepint](https://github.com/maggiepint))
+- Philipp Dunkel ([@pipobscure](https://github.com/pipobscure))
+- Matt Johnson ([@mj1856](https://github.com/mj1856))
+- Brian Terlson ([@bterlson](https://github.com/bterlson))
+- Shane Carr ([@sffc](https://github.com/sffc))
+- Ujjwal Sharma ([@ryzokuken](https://github.com/ryzokuken))
+- Philip Chimento ([@ptomato](https://github.com/ptomato))
+- Jason Williams ([@jasonwilliams](https://github.com/jasonwilliams))
+- Justin Grant ([@justingrant](https://github.com/justingrant))
 
 ## Status
 
-This proposal is currently [Stage 2](https://github.com/tc39/proposals#stage-2).
+This proposal is currently [Stage 3](https://github.com/tc39/proposals#stage-3).
+Stage 3 reviewers were: Richard Gibson, Bradley Farias, and Daniel Ehrenberg.
 
-At the time of writing, all the major design decisions that we are aware of have been made.
-The proposal is under review by the TC39 delegates and the bar for making subsequent changes is high.
-Nonetheless, changes may occur as the result of feedback from TC39 reviews and implementation in JS engines.
-
-Stage 3 Reviewers:
-
--   Richard Gibson
--   Bradley Farias
--   Daniel Ehrenberg
+This proposal is now in the hands of ECMAScript engine implementers, so the bar for making subsequent changes is extremely high.
```

Comment:
It's probably worth mentioning that the specification is still being editorially refined, and is expected to be for a while yet.

Reply:
Good idea! I added this line.

> Editorial changes to the spec and bug fixes to the spec, polyfill, tests, and docs are also ongoing, as is customary for Stage 3 proposals.